packages/web-workers/src/orchestration-steps.ts

```diff
@@ -185,6 +187,17 @@ export class OrchestrationStepsImpl implements OrchestrationSteps {
         this.logOrchestrationStep(`Completed functional tests: ${testGroupNames}`);
     }
 
+    public logTestRunStart(getTestGroupNamesFunc: () => string[] = getAllTestGroupClassNames): void {
+        const testGroupNamesStr = getTestGroupNamesFunc().reduce((prevValue, currentValue) => `${prevValue},${currentValue}`);
+        const properties = {
+            ...this.getDefaultLogProperties(),
+            functionalTestGroups: testGroupNamesStr,
+            runId: this.context.df.instanceId,
+            environment: this.availabilityTestConfig.environmentDefinition,
+        };
+        this.logger.log('Starting functional test orchestration.', LogLevel.info, properties);
```

Comment:
you could use events instead of logs for this. logs are for diagnostics purposes.